### PR TITLE
trans hnsw to hgraph in search param parse

### PR DESCRIPTION
**Pull request by LHT129**

### CODE REVIEW

**Review by wxyucs — APPROVED** [review]
lgtm

File: src/index/hgraph_zparameters.h [hunk 1]
```diff
@@ -70,4 +70,17 @@ class HGraphParameters {
     static const std::unordered_map<std::string, std::vector<std::string>> EXTERNAL_MAPPING;
 };
 
+class HGraphSearchParameters {
+public:
+    static HGraphSearchParameters
+    FromJson(const std::string& json_string);
+
+public:
+    int64_t ef_search_{30};
```
Comment (jiaweizone) [inline src/index/hgraph_zparameters.h]: public fields do not need to end with` _`

Comment (LHT129) [reply src/index/hgraph_zparameters.h]: done

**Review by jiaweizone — APPROVED** [review]
LGTM

**Pull request merged**